### PR TITLE
chore(time-to-see-data): Measure time-to-see-data in dashboards better

### PR DESCRIPTION
**Pull request by macobo**
We didnt have a measure for how long initial load _actually_ took due to refreshes that could be triggered after loading data from API. Now we do!

### CODE REVIEW

File: frontend/src/scenes/dashboard/dashboardLogic.tsx [hunk 1]
```diff
@@ -137,33 +144,19 @@ export const dashboardLogic = kea<dashboardLogicType>({
                         return null
                     }
 
-                    const refreshStartTime = performance.now()
                     const dashboardQueryId = uuid()
+                    actions.setInitialLoadTimer(action, dashboardQueryId)
```
Comment (pauldambra) [inline frontend/src/scenes/dashboard/dashboardLogic.tsx:148]: I'm tempted to nit on the name :) I think this is `loadingDashboardItemsStarted` but not blocking

Comment (pauldambra) [reply frontend/src/scenes/dashboard/dashboardLogic.tsx:148]: Interestingly while convincing myself of the order of operations here I spotted that when loading a dashboard on the home page this is called twice but not when loading a dashboard on its own page

**Review by pauldambra — APPROVED** [review]
it's getting pretty tangly in here

But the timing tangliness is only throwing the actual tangliness into sharp relief...

File: frontend/src/scenes/dashboard/dashboardLogic.tsx (hunk 1 above)
Comment (macobo) [reply frontend/src/scenes/dashboard/dashboardLogic.tsx:148]: Facinating. I'll take a look at home page separately!

**Pull request merged**